packages/notebook-extension/schema/tracker.json

```diff
@@ -734,6 +734,12 @@
           "type": "boolean",
           "default": false,
           "title": "Show Trailing Space"
+        },
+        "splitStdinHistoryBySession": {
+          "type": "boolean",
+          "default": false,
+          "title": "Split Stdin History by Kernel",
+          "description": "When enabled, line history for stdin (eg the ipdb prompt) is kept separately for different kernel sessions."
```

Comment:
I think we should go for a more future-proof `inputHistoryScope: 'global' | 'session'` (and in future 'environment') to allow extensibility. In fact we could also add 'kernel' or 'language' immediately which would group by kernel language (and could be a sensible default). I will push a commit.